pointerevents/pointerevent_attributes_mouse-manual.html

```diff
@@ -67,18 +65,11 @@
                     assert_equals(event.isPrimary, true, "isPrimary should be true");
                 }, pointerTestName + ".isPrimary attribute is correct.");
 
-                // Test width and height
-                test(function () {
-                  assert_equals(event.width, 1, "width of mouse should be 1");
```

Comment:
Should (or is) testing width/height of a mouse be done in another mouse-specific test?  Or would it be reasonable to have some pointer-type-specific test assertions here?